core/src/main/java/org/apache/accumulo/core/metadata/TabletFile.java

```diff
@@ -20,15 +20,25 @@
 
 import org.apache.hadoop.fs.Path;
 
+/**
+ * An interface that represents different types of file references that are handled by code that
+ * processes tablet files.
+ */
 public interface TabletFile {
 
   /**
-   * @return The file name of the TabletFile
+   * Returns the fileName of the TabletFile. The value return is the name itself and not the entire
+   * path.
+   *
+   * For example, if the full path for a TabletFile is
+   * 'hdfs://nn1/accumulo/tables/5a/t-0001/F0002.rf', this method returns 'F0002.rf'.
```

Comment:
You are actually right, there's probably not a reason to have getFileName(). The caller can just use tabletFile.getPath().getName().

If I drop that method should I open a new PR or just do it here?

Reply:
I would just fix the javadoc here. If you decide we don't need the other method, it can be done in a separate PR, after some experimentation with it to see what that would look like. I'm not sure if there might be undesirable consequences with that change or not.